asdf_astropy/resources/schemas/wcs/slicedwcs-1.0.0.yaml

```diff
@@ -16,7 +16,10 @@ allOf:
   - type: object
     properties:
       wcs:
-        tag: "tag:astropy.org:astropy/wcs/wcs-1*"
+        anyOf:
+          - tag: "tag:astropy.org:astropy/wcs/wcs-1*"
+          - tag: "tag:stsci.edu:gwcs/wcs-*"
```

Comment:
I assume adding this to the current schema is fine as it just extends it, but happy to make a new one.